cfgov/unprocessed/js/organisms/FormExplainer.js

```diff
@@ -0,0 +1,608 @@
+import {
```

Comment:
Copied from [OAH form explainer JS file](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/apps/owning-a-home/js/form-explainer/form-explainer.js). Main change is removal of [placeholder](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/apps/owning-a-home/js/form-explainer/form-explainer.js#L524-L570) and tab code ([here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/apps/owning-a-home/js/form-explainer/form-explainer.js#L627-L644) and [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/apps/owning-a-home/js/form-explainer/form-explainer.js#L136-L148))